File: sentimental-mario-less/mario.py
# TODO
from cs50 import get_int


h = get_int("Height: ")

while h > 8 or h < 1:
    h = get_int("Height: ")

for i in range(h):
    print(" " * (h - 1 - i), end="")
    print("#" * (i + 1))

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>
#include <ctype.h>

int main(void)
{
    int h;

    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    for (int i = 0; i < h; i++)
    {
        int s = h - i - 1;
        while (s > 0)
        {
            printf(" ");
            s--;
        }
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}
